scripts/run_locally.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -e
+
+echo "==> $(date +%H:%M:%S) ==> Setting up environment variables from sample file..."
+TEMP=temp.env \
+  && cp ../.env.sample $TEMP \
+  || exit
+
+echo "==> $(date +%H:%M:%S) ==> Using http://polygon-rpc.com as default RPC node"
+sed -i '' 's|http://url.to.node|https://polygon-rpc.com|g' $TEMP \
+  && mv $TEMP ../.env \
+  || exit
+
+echo "==> $(date +%H:%M:%S) ==> Starting up environment containers..."
+docker compose up -d \
+  && echo "==> $(date +%H:%M:%S) ==> Creating super-user for Safe Config Service..." \
+  && docker compose exec cfg-web python src/manage.py createsuperuser \
+  && echo "==> $(date +%H:%M:%S) ==> Creating super-user for Safe Transaction Service..." \
+  && docker compose exec txs-web python manage.py createsuperuser \
+  || exit
+
+echo "==> $(date +%H:%M:%S) ==> All set! You may want to add a ChainInfo into the Config service. Please use the link below to fill its data: http://localhost:8000/cfg/admin/chains/chain/add/"
```

Comment:
The third task of the issue is add ChainInfo, if you are using a default RPC for Polygon why don't add basic chainInfo for Polygon :question:

Reply:
Also, when creating superusers, is expected for the user to input the password, right?

Reply:
> The third task of the issue is add ChainInfo, if you are using a default RPC for Polygon why don't add basic chainInfo for Polygon ❓

If I recall correctly (because there was a while since 🙂 ) we thought concluded that maybe it's better to let the user enter the ChainInfo using the Django admin ([context here](https://5afe.slack.com/archives/C01GL8X8L3H/p1659542737656889)). But I'm open of you think we could make a REST call to retrieve the `ChainInfo` and inserting it (would require another PR to create the admin command in the Config Service) 🙂

> Also, when creating superusers, is expected for the user to input the password, right?

Yes, the script execution waits until the user writes the desired username/password. Is this correct?